cosmos/operators/watcher.py

```diff
@@ -184,6 +184,7 @@ def _callback(ev: EventMsg) -> None:
 
 class DbtConsumerWatcherSensor(BaseSensorOperator, DbtRunLocalOperator):  # type: ignore[misc]
     template_fields = ("model_unique_id",)  # type: ignore[operator]
+    poke_retry_number: int = 0
```

Comment:
Does poke_retry_number reset to 0 after a worker restart, or does it retain its previous value?